src/apps/code-editor/src/app/components/FileList/FileList.less

```diff
@@ -53,6 +53,8 @@
     h1 {
       color: @zesty-light-blue;
       text-transform: uppercase;
+      display: flex;
```

Comment:
This is very smelly to me. I would prefer to see a flex container wrap the h1 and tooltip separately. As opposed to converting h1 into a flex container with tooltip and text child. MUI provides quick and easy way to declare a flex container